src/test/java/org/folio/processing/events/handlers/UpdateInstanceEventHandler.java

```diff
@@ -0,0 +1,45 @@
+package org.folio.processing.events.handlers;
+
+import io.vertx.core.json.JsonObject;
+import org.folio.ActionProfile;
+import org.folio.DataImportEventPayload;
+import org.folio.processing.events.services.handler.EventHandler;
+
+import java.util.concurrent.CompletableFuture;
+
+import static org.folio.ActionProfile.Action.UPDATE;
+import static org.folio.ActionProfile.FolioRecord.INSTANCE;
+import static org.folio.DataImportEventTypes.DI_INVENTORY_INSTANCE_UPDATED;
+import static org.folio.rest.jaxrs.model.ProfileSnapshotWrapper.ContentType.ACTION_PROFILE;
+
+public class UpdateInstanceEventHandler implements EventHandler {
```

Comment:
```suggestion
public class InstanceUpdateEventHandler implements EventHandler {
```

Reply:
but it corresponds to the existing naming format of handlers 
https://github.com/folio-org/data-import-processing-core/blob/d3da666486a665c464d57d5323af9f73c6b54bd5/src/test/java/org/folio/processing/events/handlers/CreateInstanceEventHandler.java#L15